### PR TITLE
Update composer dependencies

### PR DESCRIPTION
**Pull request by carlos-mg89**
Updated the following dependencies:
- PHPUnit (to 6.5) => This implied changing the namespace for the PHPUnit tests
- PHPMailer (to latest version)
- Mockery (to latest compatible version, needs updating PHP for a more recent version)
- tpyo/amazon-s3-php-class (with latest commit and fix version to it)
- Added required PHP extensions: pdo and openssl

Reduced verbosity while executing all tests, so we can focus on seeing the result of the tests and not specific errors.

More info: https://docs.google.com/document/d/1cfJqQ1Wl_15r8iQFrRNKNmdI8zPJcQrN98oi8Kzafrc/edit?usp=sharing

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "gregwar/captcha": "^1.1",
         "geoip2/geoip2": "~2.0",
         "openclerk/country-list": "^1.0",
-        "mockery/mockery": "^0.9.4",
+        "mockery/mockery": "^1.0",
```
Comment (davidbeig) [inline composer.json:36]: Hi @carlos-mg89, I believe that in mockery 1.4 it drops support for PHP < 7.3, so I think this should be >= 1.0 and < 1.4. Maybe I'm missing something.

Thanks!

Comment (carlos-mg89) [reply composer.json:36]: Yes, that's right. I've pointed this out in my notes (GoogleDrive). However, Composer won't let you update if PHP isn't 7.3. You can try by running composer update.

But yes, we can certainly restrict this in the composer.json. I'd do it right away ;)

Comment (davidbeig) [reply composer.json:36]: True, I was missing that part!

**Comment by carlos-mg89** [comment]
@davidbeig I've finally included in this PR the upgrade from PHPUnit 5.7 to 6.5

The huge part of the changes are due to the change on the PHPUnit namespace for the test classes.

And clearly, all tests continue on green ;)

**Review by davidbeig — APPROVED** [review]

**Comment by davidbeig** [comment]
Thank you @carlos-mg89,

I have approved these changes. Really nice work! Later on I will merge them.

Seguim!

**Pull request closed without merging**